drivers/i3c/master/mipi-i3c-hci/core.c

```diff
@@ -745,6 +745,12 @@ static int i3c_hci_attach_i3c_dev(struct i3c_dev_desc *dev)
 			return ret;
 		}
 		mipi_i3c_hci_dat_v1.set_dynamic_addr(hci, ret, dev->info.dyn_addr);
+
+#ifdef CONFIG_ARCH_ASPEED
+		/* set the DEV_NACK_RETRY_CNT to max for this device */
+		mipi_i3c_hci_dat_v1.set_flags(hci, ret, W0_MASK(30, 29), 0);
```

Comment:
Are there no #define for 30 and 29?

Reply:
moved to dat_v1.c::alloc_entry() as a default flag after allocating an entry for this device.